src/Microsoft.OData.Edm/Validation/EdmErrorCode.cs

```diff
@@ -1372,5 +1372,15 @@ public enum EdmErrorCode
         /// Could not find a return on the annotated operation.
         /// </summary>
         BadUnresolvedReturn = 388,
+
+        /// <summary>
+        /// The composable escape bound function should not declare more than one.
+        /// </summary>
+        EntityComposableBoundEscapeFunctionMustBeLessOne,
+
+        /// <summary>
+        /// The non-composable escape bound function should not declare more than one.
+        /// </summary>
+        EntityNoncomposableBoundEscapeFunctionMustBeLessOne,

```

Comment:
The other member names all have assigned values; any reason not to continue that pattern?

Reply:
What i think is that to assign a value is not feasible to insert new member at middle. 

But, i am ok to add a value now.

---
In reply to: [267993749](https://github.com/OData/odata.net/pull/1400#discussion_r267993749) [](ancestors = 267993749)